### DISCUSSION TITLE
v3.7.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#9123] [Releases & Announcements]
## Changelog

- SLA for conversations (enterprise) - (alpha)
- Inbox view for notifications - (alpha)
- Reports for conversations handled by bots
- Ability to block contacts
- Conversation image improvements for agents ( zoom, preview inline images, etc )
- Export contact improvements
- Numerous bug fixes and enhancements

Thanks to @LiamAshdown, @CristianDuta, @danilofuchs, @matheus-rodrigues00  for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v3.7.0'>v3.7.0</a>.</em>

**Comment by matheus-rodrigues00** [comment]
🇧🇷🇧🇷🇧🇷

**Comment by pavlotsyhanok** [comment]
Is this yet available via helm?

**Comment by Eduardo-Skovisk** [comment]
Hello, good morning, everyone, I'm using version 3.6.0 and I'm having a problem with conversations after a while of service, being out of order, that is, recent conversations are sometimes not at the top. Has anyone else had this?

**Reply by arthurbarret0** [reply]
Yes, I reported it here:
https://github.com/chatwoot/chatwoot/issues/8765

**Comment by jmfaria2021** [comment]
I updated my selfhosted version to 3.7.0, and my inboxes were inaccessible, has anyone had this problem? what would be the solution?

**Reply by psifernandocardoso** [reply]
Same problem here too

**Reply by psifernandocardoso** [reply]
managed to solve?

**Reply by sojan-official** [reply]
did you run the migrations after updating ?

**Comment by GrygorievEvgeniy** [comment]
How to find all blocked contacts and unblock them?

**Comment by psifernandocardoso** [comment]
No. I only updated via portainer. How do I run migrations?

*Att,*
*Fernando Cardoso*


Em qua., 3 de abr. de 2024 às 13:23, Sojan Jose ***@***.***>
escreveu:
